rest_framework_inclusions/core.py

```diff
@@ -16,16 +16,23 @@ class Error(Exception):
 
 
 class InclusionLoader:
+    nested_inclusions_use_parent = True
+
     def __init__(self, allowed_paths):
         self.allowed_paths = allowed_paths
         self._seen = set()
 
+    def get_model_key(self, obj, *args, **kwargs):
+        return obj.__class__._meta.label
+
     def inclusions_dict(self, serializer):
         entries = self._inclusions((), serializer, serializer.instance)
         result = {}
         for obj, inclusion_serializer in entries:
-            model_key = obj.__class__._meta.label
-            data = inclusion_serializer(instance=obj).data
+            model_key = self.get_model_key(obj, inclusion_serializer)
+            data = inclusion_serializer(
+                instance=obj, context={"request": serializer.context.get("request")}
```

Comment:
```suggestion
                instance=obj, context=serializer.context}
```
perhaps? that's a bit more future proof and flexible to just pass along all context